flow-decomposition/src/main/java/com/powsybl/flow_decomposition/rescaler/DecomposedFlowRescalerAcerMethodology.java

```diff
@@ -34,24 +33,19 @@ private static double rescaleValue(double initialValue, double delta, double sum
     }
 
     @Override
-    public DecomposedFlow rescale(DecomposedFlow decomposedFlow) {
-        double acTerminal1ReferenceFlow = decomposedFlow.getAcTerminal1ReferenceFlow();
-        double acTerminal2ReferenceFlow = decomposedFlow.getAcTerminal2ReferenceFlow();
-        if (Double.isNaN(acTerminal1ReferenceFlow) || Double.isNaN(acTerminal2ReferenceFlow)) {
-            return decomposedFlow;
-        }
+    protected boolean shouldRescaleFlows(DecomposedFlow decomposedFlow) {
+        // - if AC flows are not NaN
+        return !Double.isNaN(decomposedFlow.getAcTerminal1ReferenceFlow()) && !Double.isNaN(decomposedFlow.getAcTerminal2ReferenceFlow());
```

Comment:
`Double.isFinite` would be more readable so you can remove the comment

Reply:
Extract a static method `hasFiniteAcFlowsOnEachTerminal` to share it with other `DecomposedFlowRescalers`.